templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -45,7 +45,7 @@ objects:
   kind: ServiceAccount
   metadata:
     name: asb
-    namespace: ansible-service-broker
+    namespace: "${NAMESPACE}"
```

Comment:
is there a catasb related PR to allow overriding this? I mean most of the variables have a corresponding value in `all.yml`.

Reply:
these scripts would be outside of catasb. these are essentially a 'striped down' versions of the catasb functionality.